integration/e2e/pages/viewer.po.ts

```diff
@@ -341,6 +351,42 @@ export class ViewerPage {
     return Math.round(svgParentDimensions.height) === Math.round(overlayDimensions.height);
   }
 
+  async sendKeyboardEvent(key: string): Promise<void> {
+    let iKey: string = null;
+    if (key === 'PageDown') {
+      iKey = Key.PAGE_DOWN;
+    } else if (key === 'ArrowRight') {
+      iKey = Key.ARROW_RIGHT;
+    } else if (key === 'n') {
+      iKey = Key.chord('n');
+    } else if (key === 'PageUp') {
+      iKey = Key.PAGE_UP;
+    } else if (key === 'ArrowLeft') {
+      iKey = Key.ARROW_LEFT;
+    } else if (key === 'p') {
+      iKey = Key.chord('p');
+    } else if (key === 'Home') {
+      iKey = Key.HOME;
+    } else if (key === 'End') {
+      iKey = Key.END;
+    } else if (key === '+') {
+      iKey = Key.ADD;
+    } else if (key === '-') {
+      iKey = Key.SUBTRACT;
+    } else if (key === '0') {
+      iKey = Key.chord('0');
+    } else if (key === 's') {
+      iKey = Key.chord('s');
+    } else if (key === 'c') {
+      iKey = Key.chord('c');
+    } else if (key === 'Esc') {
+      iKey = Key.ESCAPE;
+    }
+
+    await browser.actions().sendKeys(iKey).perform();
+    return await browser.sleep(await this.getAnimationTime() * 1000);
```

Comment:
Rename getAnimationTime to getAnimationTimeInSec. And create a getAnimationTimeInMs